src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -2,8 +2,11 @@
 
 import java.util.Optional;
 import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.model.DataProcessingException;
 import mate.academy.hibernate.relations.model.Movie;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
 public class MovieDaoImpl extends AbstractDao implements MovieDao {
```

Comment:
Make sure that the exception messages are specific to the `MovieDaoImpl` class. The message in `DataProcessingException` should clearly identify that the error is occurring in the movie DAO layer.